### DISCUSSION TITLE
Library Categories and Series

### DISCUSSION THREAD

**Discussion by solveitconsultants** [TortugaPower/BookPlayer#617] [Ideas]
# 🗣 Library Organization 

Just a shout out to the developers, you are doing a great job. Every progression so far has been fantastic!

Something I would love to see is the ability to use categories, tags, or series to manage my larger libraries when I am on the road.

I have a lot of series (Asterix and Obelix for example) which have episodes (Same with Alfred Hitchcocks Three Detectives) where a series name, episode number, and even category "Children's Books"  for example would be beneficial.

Integration of a search function would be nice but categories, tags, series, and episode information would make this likely less important.

It would be great if the system could sync with MusicBee's meta information.

Regardless keep up the good work.

**Comment by pichfl** [comment]
We are working on extended support for nested folders and structures as well as extended meta data.

4.0 brought a big stepping stone for this with support for importing folders and a more solid file structure, so you'll see progress in the future. 

The Library UI will get an overhaul soon (#538 is a start) and more filters and search options are definitely a good idea 👍
